<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMaterialInput</class>
 <widget class="QDialog" name="DialogMaterialInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>物料投料</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="labelPackageBatch">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>包装条码：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboStationID">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelStationID">
       <property name="text">
        <string>工位编号：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelMaterialBatch">
       <property name="text">
        <string>物料批次：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editBatchQuantity">
       <property name="placeholderText">
        <string>请扫描物料数量</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelBatchQuantity">
       <property name="text">
        <string>批次数量：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editMaterialBatch">
       <property name="placeholderText">
        <string>请扫描物料批次</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editPackageBatch">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>请扫描物料最小包装条码</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboMaterialID">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelStationID_2">
       <property name="text">
        <string>物料编号：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>上料</string>
       </property>
       <property name="icon">
        <iconset resource="../PCS.qrc">
         <normaloff>:/images/pngs/common_accept.png</normaloff>:/images/pngs/common_accept.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
       <property name="icon">
        <iconset resource="../PCS.qrc">
         <normaloff>:/images/pngs/common_reject.png</normaloff>:/images/pngs/common_reject.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboStationID</tabstop>
  <tabstop>comboMaterialID</tabstop>
  <tabstop>editMaterialBatch</tabstop>
  <tabstop>editBatchQuantity</tabstop>
  <tabstop>editPackageBatch</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../PCS.qrc"/>
 </resources>
 <connections/>
</ui>
